app/views/core_induction_programme/lesson_parts/show_split.html.erb

```diff
@@ -0,0 +1,57 @@
+<% content_for :outside_container do %>
+<div class="govuk-width-container govuk-main-wrapper govuk-body">
+  <%= govuk_breadcrumbs breadcrumbs: course_lesson_breadcrumbs(@current_user, @course_lesson_part.course_lesson) %>
+  <h1 class="govuk-heading-l">Split lesson part</h1>
+  <ul>
+    <li>
+      The part on the left hand side is the one you are splitting (current).
```

Comment:
this is very very inaccessible

Reply:
Eh, it's meant to be internal. 
You mean the list is inaccessible? Or left/right distinction?

Reply:
left/right distinction! both the words and the actual layout

Reply:
Well, truth be told I think there is going to be one unlucky user of that page, which is Mark A. 

I don't think there is a very accessible way of handling this problem to begin with. The problem being 'splitting very long markdown roughly in half, in a text area, and showing what the outcome will look like'.

The amount of content to show is massive, so much so that showing it in a column, one after another, looked really unintuitive. That's why I opted to use visuals to show them in parallel.

I can rephrase it to be "The first part is the one you are splitting, the second part is new" or something like that if you think it's much better.

Reply:
if it's for internal usage only I'm not sure it matters that much :)

Reply:
Alright :)